index.html

```diff
@@ -418,9 +418,11 @@ <h2>JOSE Header Parameters and JWT Claims</h2>
         secured and semantics provided by the [[VC-DATA-MODEL-2.0]]. This means that while the security
         features of this specification ensure data integrity and authenticity, they do
         not dictate the interpretation of claim data.
-        
-        Implementers are RECOMMENDED to avoid conflicting values, especially with claims such as
-        `issuer` with `iss`, `id` with `jti`, and `credentialSubject.id` with `sub`.
+      </p>
+      <p>
+        Implementers SHOULD avoid setting JWT claims to values that conflict with
```

Comment:
SHOULD -> MUST
otherwise if an implementation does set conflicting values what does this mean?

Reply:
It means that implementer (of Issuer and/or Holder/Presenter software) SHOULD understand that they're introducing inconsistencies to their data, and that downstream implementers (of Holder/Receiver and/or Verifier software) MAY reject such JWTs and their enclosed VCs.

Reply:
@David-Chadwick responding here. I agree with @TallTed's interpretation.

If @selfissued also believes this language is better as a MUST I will make a revision PR.